### ISSUE TITLE
customElement.define() within constructor/setup methods produce an error on consequent calls

### ISSUE THREAD

**Issue by Nowan** [retejs/lit-plugin#7]
### Describe the bug

First of all, really great job with the lib, kudos to the team!

Now, say we have multiple rete editors in our app and would like to configure them differently. Or just one but removing and reinstantiating it once user gets into/leaves the router page with the editor. 

There are `customElement.define()` methods calls being present within:
 - `LitPlugin` constructor
 - `ClassicPreset.setup`
 - `ContextMenuPreset.setup`
 - `MinimapPreset.setup`
 - `ReroutePreset.setup`

Resulting that consequent call to any of those method throws
> NotSupportedError: Failed to execute 'define' on 'CustomElementRegistry': the name "rete-root" has already been used with this registry

### How to reproduce

- Initialize LitPlugin twice:
```javascript 
import { LitPlugin, Presets } from "@retejs/lit-plugin";

function createLitRenderer() {
  const render = new LitPlugin();
  
  render.addPreset(Presets.classic.setup());
  render.addPreset(Presets.contextMenu.setup());
  render.addPreset(Presets.minimap.setup());
  render.addPreset(Presets.reroute.setup());
  
  return render;
}

const rendererA = createLitRenderer();
const rendererB = createLitRenderer();
```

### Expected behavior

The following error is not thrown:
> NotSupportedError: Failed to execute 'define' on 'CustomElementRegistry': the name "..." has already been used with this registry

### Dependencies

@babel/plugin-proposal-class-properties@^7.18.6
@babel/plugin-proposal-decorators@^7.24.7
globals@^15.9.0
lit@^3.1.4
rete-area-plugin@^2.0.0
rete-cli@~2.0.1
rete-render-utils@^2.0.2
rete@^2.0.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Ni55aN** [comment]
> 🎉 This PR is included in version 2.0.7 🎉
> 
> The release is available on:
> 
> * [GitHub release](https://github.com/retejs/lit-plugin/releases/tag/v2.0.7)
> * [npm package (@latest dist-tag)](https://www.npmjs.com/package/@retejs/lit-plugin/v/2.0.7)
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀

released

**Comment by Ni55aN** [comment]
thanks for the contribution!
